tests/material_derivatives_simpler.cc

```diff
@@ -0,0 +1,95 @@
+// make sure we use the additional material model outputs
+#include <aspect/simulator.h>
+#include <deal.II/grid/tria.h>
+#include <aspect/material_model/interface.h>
+#include <aspect/material_model/simple.h>
+#include <aspect/simulator_access.h>
+
+#include <iostream>
+#include <typeinfo>
+
+
+int f()
+{
+  const int dim=2;
+std::cout << "Flag 1" << std::endl;
+  using namespace aspect::MaterialModel;
+  MaterialModelInputs<dim> in_basis(3,3);
+  in_basis.composition[0][0] = 1;
+  in_basis.composition[0][1] = 0;
+  in_basis.composition[0][2] = 0;
+  in_basis.composition[1][0] = 0.5;
+  in_basis.composition[1][1] = 0.25;
+  in_basis.composition[1][2] = 0.25;
+  in_basis.composition[2][0] = 0;
+  in_basis.composition[2][1] = 0.5;
+  in_basis.composition[2][2] = 0.5;
+
+  in_basis.pressure[0] = 10;
+  in_basis.pressure[1] = 1000;
+  in_basis.pressure[2] = 1e6;
+
+  in_basis.strain_rate[0] = SymmetricTensor<2,dim>(); // What do I fill it with?
+  in_basis.strain_rate[1] = SymmetricTensor<2,dim>(); // What do I fill it with?
+  in_basis.strain_rate[2] = SymmetricTensor<2,dim>(); // What do I fill it with?
```

Comment:
I think this is reasonable. Or, and maybe better, just choose a nonzero tensor with entries that are about the order of magnitude we would expect in the earth.